README.md

```diff
@@ -2,6 +2,40 @@
 
 Enables validation for inputs in conjunction with hyper-validator modules.
 
+## Usage
+
+This is the base for all the validator modules. It collects the different validator modules and runs the
+validattions at the right time. To install it simply do `npm install hyper-validator-base --save-dev`.
+This will add the validator base in you node_modules folder and insert it as a dependecy in `package.json`.
```

Comment:
And is there a reason you have “package.json” in code formatting but not “node_modules”? Also, should it be code formatted at all? It's not code, it's a directory. I prefer using _emphasized text_ for files and directories, or nothing at all.